canonical/ubuntu.com
# Update /cloud with `list-stepped`

Replace `.stepped` and `.list-step` usage with new `.list-stepped`.

Also remove `autopilot-b` page - not used anywhere.
## QA
- `make run`
- Check http://localhost:8001/cloud/openstack/managed-cloud against [live](https://www.ubuntu.com/cloud/openstack/managed-cloud)
- Check http://localhost:8001/cloud/openstack/autopilot against [live](http://www.ubuntu.com/cloud/openstack/autopilot)
